src/components/Carousel/index.tsx

```diff
@@ -69,11 +66,9 @@ const ContentCarousel: FC<Props> = ({ children, ...props }: Props) => (
       responsive={sizing}
       swipeable
       draggable
-      infinite
-      centerMode
       arrows
```

Comment:
Carousel left/right arrows differ in design

![Screenshot 2024-02-21 at 11 37 15 AM](https://github.com/CBIIT/crdc-datahub-ui/assets/70384003/e26084ac-5c29-402b-8536-9384699dfcf1)
![Screenshot 2024-02-21 at 11 37 53 AM](https://github.com/CBIIT/crdc-datahub-ui/assets/70384003/de235381-44bf-458c-b113-c479aa544784)

Reply:
Completely missed this, thanks for catching it. Updated to use the existing chevrons in 4fb4c3c.